js/views/chatConversationVw.js

```diff
@@ -0,0 +1,165 @@
+var Backbone = require('backbone'),
+  $ = require('jquery'),
+  moment = require('moment'),
+  app = require('../App.js').getApp(),
+  loadTemplate = require('../utils/loadTemplate'),
+  baseVw = require('./baseVw'),
+  ChatMessageVw = require('./chatMessageVw');
+
+module.exports = baseVw.extend({
+  className: 'chatConversation',
+
+  events: {
+    'click .js-closeConversation': 'onClickClose',
+    'keydown textarea': 'onKeydownMessage',
+    'keyup textarea': 'onKeyupMessage',
+    'click .js-conversationSettings': 'toggleConvoSettings',
+    'click .chatConversationMenu a': 'closeConvoSettings',
+    'click .js-blockUser a': 'onBlockClick'
```

Comment:
This is very minor, but it would be better to put the classes directly on the anchor element, that way if they were switched to some other element (like a div or span) this code wouldn't need to be changed.